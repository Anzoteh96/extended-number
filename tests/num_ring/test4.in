439840841343
352667947302
969330415781
