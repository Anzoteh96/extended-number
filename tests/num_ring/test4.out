Enter the number base b:
Enter two numbers n and m:
b is 439840841343
n mod b is 352667947302
m mod b is 89648733095
n + m is 2475839054
n - m is 263019214207
n * m is 65960876241
n ** m is 11981907357
